packages/nimble-components/src/text-field/styles.ts

```diff
@@ -118,7 +117,11 @@ export const styles = css`
 
     slot[name='start']::slotted(*) {
```

Comment:
We have a bunch of other controls that have a "start icon next to some text", i.e. various buttons (button, anchor button, menu button), menu items, table headers, mapping column, tree item, tooltip and banner (fixed severity icons).
Do we need to drive any consistency around those?